kong/cmd/init.lua

```diff
@@ -1,6 +1,6 @@
 require("kong.core.globalpatches")({cli = true})
 
-local prng_seed = math.randomseed()
+math.randomseed() -- Generate PRNG seed
```

Comment:
There is really no point in adding comments that just explain what a (well-known) function title already does.